orc-rt/include/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+set(ORC_RT_HEADERS
+    orc-rt-c/orc-rt.h
+)
+
+# TODO: Switch to filesets when we move to cmake-3.23.
+add_library(orc-rt-headers INTERFACE)
+target_include_directories(orc-rt-headers INTERFACE
+    $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
+    $<INSTALL_INTERFACE:include>
+)
+set_property(TARGET orc-rt-headers
+    PROPERTY PUBLIC_HEADER ${ORC_RT_HEADERS}
+)
+install(TARGETS orc-rt-headers
+    DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/
+    COMPONENT OrcRT_Development
+)
```

Comment:
Have you tested this on non-Apple platforms? You need to set `PUBLIC_HEADER` option with `install(TARGET ...)` to have it install headers specified by the `PUBLIC_HEADER` property, see https://cmake.org/cmake/help/latest/command/install.html#targets.

We usually use explicit `install(FILES ...)` elsewhere in LLVM for consistent behavior across platforms, see https://github.com/llvm/llvm-project/blob/5b5589978167ab7abc6a5e8a3a1ce7d8487ce73a/libcxx/include/CMakeLists.txt#L1726-1733